drivers/123_open/util.go

```diff
@@ -0,0 +1,196 @@
+package _123_open
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/OpenListTeam/OpenList/drivers/base"
+	"github.com/go-resty/resty/v2"
+	log "github.com/sirupsen/logrus"
+)
+
+var ( //不同情况下获取的AccessTokenQPS限制不同 如下模块化易于拓展
+	Api = "https://open-api.123pan.com"
+
+	AccessToken    = InitApiInfo(Api+"/api/v1/access_token", 1)
```

Comment:
增加 Refresh Token API 地址。

```suggestion
	AccessToken    = InitApiInfo(Api+"/api/v1/access_token", 1)
	RefreshToken   = InitApiInfo(Api+"/api/v1/oauth2/access_token", 1)
```